pkg/model/rhsso_deployment.go

```diff
@@ -90,6 +90,10 @@ func getRHSSOEnv(cr *v1alpha1.Keycloak, dbSecret *v1.Secret) []v1.EnvVar {
 			Name:  "X509_CA_BUNDLE",
 			Value: "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
 		},
+		{
+			Name:  "PROXY_ADDRESS_FORWARDING",
```

Comment:
From the Jira, it looks like this is not necessary for RHSSO. I haven't found any usage of this variable in our image, however, it is maybe necessary for the underlying container. @iankko @hmlnarik Do you know, whether it is necessary to add this ENV for RHSSO?

Reply:
> From the Jira, it looks like this is not necessary for RHSSO. I haven't found any usage of this variable in our image, however, it is maybe necessary for the underlying container. @iankko @hmlnarik Do you know, whether it is necessary to add this ENV for RHSSO?

@mhajas It's (the `X509_CA_BUNDLE` variable) is needed just for the `*-x509-*` templates, or for the image itself only in the case, you want the RH-SSO https & Jgroups keystores & the RH-SSO truststore to be autogenerated. The code in question is [here](https://github.com/jboss-container-images/redhat-sso-7-openshift-image/blob/sso74-dev/modules/sso/config/launch/setup/74/added/launch/openshift-x509.sh#L81) (there's corresponding counterpart for the RH-SSO 7.3 for OpenShift image too).

If the `X509_CA_BUNDLE` variable isn't defined, the image expects the user provide the definition of the RH-SSO https & JGroups keystores and the truststore. For non `*-x509-*` templates, this is done by setting corresponding necessary variables within them. [Here's the image message](https://github.com/jboss-container-images/redhat-sso-7-openshift-image/blob/sso74-dev/templates/sso74-https.json#L22), detailing which of them need to be set the keystores & the trustore to work properly.

If it's not set at all (neither the `X509_CA_BUNDLE` variable, nor the `*_SECRET` variables are set), the SSL / TLS won't work for the image (the pod launch will fail with message about the impossibility to set up the keystores & truststore). Unsure if for the operator, the pre-creation of the keystores & truststores is handle somehow (@slaskawi do you know?) If there's such substitute functionality, we could probably use that one. If there isn't, we would need to utilize the `X509_CA_BUNDLE` (possibly allowing the user to specify custom CAs if necessary) variable as it's done in the images currently.

HTH, Jan

Reply:
@iankko Thanks for the explanation. Sorry I didn't mention it in my comment, but I was asking about the `PROXY_ADDRESS_FORWARDING` variable instead of `X509_CA_BUNDLE`.

Reply:
That's a good point. It's set to true by default (I mean the setting in the Undertow subsystem).

Reply:
Removed.